build.gradle.kts

```diff
@@ -73,6 +73,7 @@ tasks.register("checkAll") {
     dependsOn(
         "ktlintCheckAll",
         "detektAll",
+        "lintCheckAll",
```

Comment:
why are we adding this ? We are not using _ktlint_ for this ?

Reply:
ktLint is a lint for the Kotlin language
lint is a lint on the Android framework.

Reply:
Reason why I asked is that I thought _ktlint_ is doing also what lint was supposed to do + linting Kotlin code.

Reply:
KtLint is specific to Kotlin, it doesn't know anything about Android. It can be used for people writing CLI apps or server side code in Kotlin too.